src/accel/SharedParams.cc

```diff
@@ -258,6 +250,20 @@ SharedParams::SharedParams(SetupOptions const& options)
         return;
     }
 
+    if (mode_ == Mode::kill_offload)
+    {
+        //! \todo: Set up default or all supported particles?
+        particles_ = IS::default_offload_particles();
+    }
+
+    if (mode_ == Mode::enabled)
+    {
+        // Set up offloaded particles based on user input
+        auto const& user_offload = options.offload_particles;
+        particles_ = user_offload.empty() ? IS::default_offload_particles()
+                                          : user_offload;
```

Comment:
```suggestion
    if (mode_ == Mode::enabled || mode_ == Mode::kill_offload)
    {
        // Set up offloaded/killed particles based on user input
        auto const& user_offload = options.offload_particles;
        particles_ = user_offload.empty() ? IS::default_offload_particles()
                                          : user_offload;
```

Also maybe SharedParams is a better place to store the hardcoded "supported particles" vetors? We were originally doing that at one point, and it seems like a more accessible+appropriate place to get the list of default+supported.